src/torch/12-deeprank-train/deeprank-train.lua

```diff
@@ -26,7 +27,7 @@ function getSimilarityModel()
   --leftImage:add(nn.SpatialConvolutionMM(512, 126, 1, 1, 1, 1, 0, 0):cuda())
```

Comment:
Should we also not kill lines which are commented out, and which are not actual comments?